src/Schema/PostgreSQLSchemaManager.php

```diff
@@ -529,7 +529,7 @@ protected function _getPortableTableColumnDefinition($tableColumn)
             $column->setPlatformOption('collation', $tableColumn['collation']);
         }
 
-        if ($column->getType()->getName() === Types::JSON) {
+        if ($column->getType() instanceof JsonType) {
```

Comment:
I wonder if this should be considered a BC-break? If somebody reimplements `JsonType` without extending it, then overrides it with the same name, it would be.

Reply:
This code stinks.  I don't think there's a way to change this code w/o breaking something, so it's better not to change it.

The problem is that the new code has the same issue.

Reply:
`instanceof` seems as a wrong direction to me, as any overriden types extending Type class directly will fail this condition

Reply:
@morozov maybe a better condition would be `if isset($jsonb) {`? Right now, it seems that if somebody mistakenly specifies the `jsonb` option on, say, an integer column, this piece of code will silently discard it. I don't think setting that option would have any effect, since it's only used here:

https://github.com/doctrine/dbal/blob/141daa53e4f717a9ca234182b81179f9d1a573d5/src/Platforms/PostgreSQLPlatform.php#L1217-L1224

@mvorisek , that's precisely what I pointed out above already, I don't need you to parrot my sentences.